docs/docs-beta/docs/guides/declarative-automation.md

```diff
@@ -1,4 +1,136 @@
+sidebar_position: 10
 sidebar_label: "Declarative automation"
----
\ No newline at end of file
+---
+
+# Declarative Automation
+
+TODO: ADD EXPERIMENTAL CALLOUT
+
+Dagster can automatically materialize assets when criteria are met, enabling a declarative approach to asset materialization. Instead of defining explicit workflows to materialize assets, you describe the conditions under which they should be materialized and let the system kick off runs in response.
+
+For example, you have an asset that's scheduled to execute every day at midnight. Instead of executing whether there's new data or not, you can use Declarative Automation to materialize the asset only after its parent has been updated.
+
+Using Declarative Automation helps you:
+
+- Ensure you're working with the most up-to-date data
+- Optimize resource usage by only materializing assets when needed
+- Simplify how your team understands their assets by consolidating all asset logic to a single location
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need familiarity with:
+
+- [Assets](/concepts/assets)
+- [Sensors](/todo)
+- [Code locations](/todo)
+
+</details>
+
+## Define automation conditions
+
+Automation conditions describe the conditions under which an asset should be executed. They can be set on native Dagster assets (`@asset`) or [external assets](/guides/external-assets) (`AssetSpec`).
+
+```python
+from dagster import dg
+
+# Condition that materializes an asset that's never been materialized, or
+# when the asset's parents update, as long as they're not missing or currently updating
+@dg.asset(automation_condition=dg.AutomationCondition.eager())
+def my_eager_asset(): ...
+
+# Condition that materializes an asset once per cron schedule tick
+dg.AssetSpec("my_cron_asset", automation_condition=dg.AutomationCondition.on_cron("@daily"))
+```
+
+When automation conditions for an asset are met, a [sensor](/guides/sensors) will kick off a run to materialize the asset. This sensor, named `default_automation_condition_sensor`, will be available for each code location and monitor all assets within that location.
+
+To use multiple sensors or change the properties of the default sensor, refer to the `AutomationConditionSensorDefinition` documentation.
+
+TODO: Add something here about the operand reference
+
+:::tip
+By default, sensors aren't enabled when first deployed to a Dagster instance.
+In the Dagster UI, click **Automation > Sensors** to find and enable the sensor.
+:::
+
+## Conditions targeting dependencies
+
+Upstream assets commonly influence downstream materialization decisions. To create automation conditions that target dependencies, use the `AutomationCondition.any_deps_match()` operator. This operator takes an arbitrary `AutomationCondition`, applies it to each upstream asset, and then maps the results to the corresponding downstream partitions.
```

Comment:
This is 'explaining the machinery' rather than helping me understand what's going on.
> This operator takes an arbitrary `AutomationCondition`, applies it to each upstream asset, and then maps the results to the corresponding downstream partitions.
Not sure what this ...actually does